website_v2/gcloud-powershell-website-v2/www/static/app/tableController.js

```diff
@@ -0,0 +1,28 @@
+﻿var app = angular.module('powershellSite');
+
+/** The controller for the table of contents. **/
+app.controller('TableCtrl', function($scope, $attrs) {
+    /** Whether or not a product's information is expanded. **/
+    this.expanded = false;
+    /** What the current active product is. **/
+    this.activeProduct = '';
+
+    /** clickProduct is used when a product is clicked. **/
+    this.clickProduct = function(name) {
```

Comment:
I would define this in a class instead of all in a function like you did, not required though